### PR TITLE
Add rancher selinux policies as standalone package

### PR DESCRIPTION
**Pull request by mudler**
https://github.com/ibuildthecloud/os2/blob/master/Dockerfile#L69

Signed-off-by: Ettore Di Giacinto <edigiacinto@suse.com>

### CODE REVIEW

File: packages/selinux/build.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+{{if eq .Values.distribution "opensuse" }}
+prelude:
+- zypper ar https://download.opensuse.org/repositories/security:/SELinux/openSUSE_Leap_15.3/security:SELinux.repo
+- zypper --gpg-auto-import-keys in -y --allow-vendor-change --allow-downgrade container-selinux -libsemanage1
+
+steps:
+- |
+    mkdir /tmp/rpm && \
+    cd /tmp/rpm && \
+    curl -L -O https://github.com/k3s-io/k3s-selinux/releases/download/v0.3.testing.0/k3s-selinux-0.3-0.el7.noarch.rpm && \
```
Comment (Itxaka) [inline packages/selinux/build.yaml:16]: do we have checksums for this packages somehow?

**Review by Itxaka — APPROVED** [review]

File: packages/selinux/build.yaml (hunk 1 above)
Comment (mudler) [reply packages/selinux/build.yaml:16]: Good point! we do actually https://github.com/k3s-io/k3s-selinux/releases/download/v0.3.stable.0/sha256sum-centos7-noarch.txt

Comment (mudler) [reply packages/selinux/build.yaml]: done! I've also split the packages separately

**Review by Itxaka — APPROVED** [review]
nice!

**Comment by mudler** [comment]
All required tests seems to be passed, merging :+1:

**Pull request merged**